frontend/apps/reader/modules/readerfont.lua

```diff
@@ -822,20 +844,34 @@ function ReaderFont:sortFaceList(face_list)
             return seen_fonts[t[i]]
         end)
     end
+    return face_list
+end
+
+function ReaderFont:sortFaceList(face_list)
+    -- Return the given list of font faces, but with newly added fonts first.
+    -- Potentially return list sorted by recently set, per config
+    -- and hide most fallback fonts, per config
```

Comment:
I don't understand the last line of this comment: "hide most fallback fonts, per config":
- what are these fallback fonts? All ours? The Noto* fonts that _you_ consider fallbacks ?
- do we hide them?
- which "per config"?

Reply:
Oops. This is a copy-paste error from an earlier attempt. I initially was taking the list of fonts from the hardcoded `CreDocument.fallback_fonts ` table. But Noto Sans Arabic UI isn't there, and I thought it was cleaner to just pull all the Noto fonts.

Line 853 here should be deleted.